### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by AbelGebrekiros**
*added section about local development to the readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,4 +9,7 @@ Git is officially defined as a *distributed version control system* (VCS).
 
 In other words, it's a system that tracks changes to our project files over time. It enables us to record project changes and go back to a specific version of the tracked files, at any given point in time.  This system can be used by many people to efficiently work together and **collaborate on team projects**, where each developer can have their own version of the project, distributed on their computer. Later on, these individual versions of the project can be merged and adapted into the main version of the project.
 
-Basically, it's a massively popular tool for coordinating parallel work and managing projects among individuals and teams. Needless to say, knowing how to use Git is one of the most important skills for any developer nowadays - and it's definitely a great addition to your resume!
\ No newline at end of file
+Basically, it's a massively popular tool for coordinating parallel work and managing projects among individuals and teams. Needless to say, knowing how to use Git is one of the most important skills for any developer nowadays - and it's definitely a great addition to your resume!
+
+## Local Development
```
Comment (AbelGebrekiros) [inline README.md:14]: added new sub header
